src/helpers/linkQueries.ts

```diff
@@ -0,0 +1,65 @@
+import { Link, linkModel } from "../schemas/link"
+import Fuse from "fuse.js";
+import { SelectMenuInteraction } from "discord.js";
+import { Types } from "mongoose";
+
+var localLinks: Link[];
+
+export const GetAllShortenLinks = async (): Promise<Link[]> => {
+    let allLinks: Link[] = await linkModel.find()
+    //fs.writeFileSync(path.resolve(__dirname, "./links.json"), JSON.stringify(allLinks));
+    initLocalLinks(allLinks)
+    return allLinks;
+}
+
+const initLocalLinks = (allLinks: Link[]) => {
```

Comment:
What exactly is the purpose of this? Why not just do the assignment where needed?